src/main/java/org/jabref/gui/preferences/external/ExternalTab.java

```diff
@@ -77,6 +83,9 @@ public void initialize() {
             validationVisualizer.initVisualization(viewModel.terminalCommandValidationStatus(), customTerminalCommand);
             validationVisualizer.initVisualization(viewModel.fileBrowserCommandValidationStatus(), customFileBrowserCommand);
         });
+
+        ActionFactory actionFactory = new ActionFactory(Globals.getKeyPrefs());
```

Comment:
We want to get rid of Globals, because it creates a direct dependency. So better use the         preferencesService.getKeyBindingRepository()

Reply:
Fixed